Dockerfile

```diff
@@ -18,7 +19,7 @@ ENV ANALYTICS_API_CFG /edx/etc/analytics_api.yml
 
 WORKDIR /edx/app/analytics_api
 COPY requirements /edx/app/analytics_api/requirements
-RUN pip3 install -r requirements/production.txt
+RUN python3.8 -m pip install -r requirements/production.txt
```

Comment:
With the environment variables set above to activate the virtualenv, this can just be the simpler `RUN pip install ...` (although what's here now should work also).